source/javascript/src/daily.js

```diff
@@ -119,6 +122,46 @@ collapse.addEventListener('click', () => {
   }
 })
 
+tagOptions.addEventListener('click', (event) => {
+  const collectionName = event.target.textContent
+  document.querySelector('.tags').append(new Tag(collectionName))
+  addCollectionTag(collectionName)
+})
+
+function addCollectionTag (collectionName) {
+  wrapper.transaction((event) => {
+    const db = event.target.result
+
+    const transaction = db.transaction(['currentLogStore'], 'readwrite')
+    const objectStore = transaction.objectStore('currentLogStore')
+    objectStore.openCursor().onsuccess = function (event) {
+      const cursor = event.target.result
+
+      if (cursor) {
+        const json = cursor.value
+        const collections = json.properties.collections
+        const currentLog = json.current_log
```

Comment:
@will-chung  `current_log` and `current_collection` are going to be refactored/replaced soon; just a heads-up in cause you want to help me with these changes over the next two days

Reply:
I think it's a good idea

Reply:
yeah we should definitely have a session soon to update all routing